### PR TITLE
Improve readme clarity

### PR DESCRIPTION
**Pull request by jul-sh**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -12,11 +12,11 @@ GitHub allows you to [co-author commits](https://help.github.com/articles/creati
 
 ### Flags
 
-You can also utilize git flags with co-commit: eg `git commit --all` becomes `npx co-commit --all`. Additionally, you can skip prompts using flags: eg: `-m "your commit message"` or `-co "mariiapunda, juliettepretot"`.
+You can also utilize git flags with co-commit: eg `git commit --all` becomes `npx co-commit --all`. Additionally, you can skip prompts using flags: eg: `-m "your commit message"` or `-co "mariiapunda"`.
 
 ### Preinstall co-commit
 
-Each time you run co-commit, `npx` will install the package. This will usually take around one second. To skip this you can preinstall co-commit using `npm i -g co-commit`.
+Each time you run co-commit, `npx` will install the package. This will usually only takes a few seconds. To skip this you can preinstall co-commit using `npm i -g co-commit`.
```
Comment (jul-sh) [inline README.MD:20]: It's like 3 sec in the gif, sooo 🙈

File: README.MD [hunk 2]
```diff
@@ -10,13 +10,13 @@ GitHub allows you to [co-author commits](https://help.github.com/articles/creati
 
 ## 🧠 Advanced
 
-### Flags
+### Make it blazing fast 🔥
```
Comment (Tom-Bonnike) [inline README.MD:13]: :awesome:

File: README.MD [hunk 3]
```diff
@@ -10,13 +10,13 @@ GitHub allows you to [co-author commits](https://help.github.com/articles/creati
...
 
-You can also utilize git flags with co-commit: eg `git commit --all` becomes `npx co-commit --all`. Additionally, you can skip prompts using flags: eg: `-m "your commit message"` or `-co "mariiapunda, juliettepretot"`.
+Each time you run co-commit, `npx` will install the package. This will usually only takes a few seconds. To skip this you can preinstall co-commit using `npm i -g co-commit`.
 
-### Preinstall co-commit
+### CLI arguments
 
-Each time you run co-commit, `npx` will install the package. This will usually take around one second. To skip this you can preinstall co-commit using `npm i -g co-commit`.
+You can also utilize git flags with co-commit: eg `git commit --all` becomes `npx co-commit --all`. Additionally, you can skip prompts using flags: eg: `-m "your commit message"` or `-co "mariiapunda"`.
```
Comment (Tom-Bonnike) [inline README.MD:19]: `npx co-commit mariiapunda` would actually be awesome UX

File: README.MD (hunk 1 above)
Comment (Tom-Bonnike) [reply README.MD:20]: blazing slow 🐌

**Review by Tom-Bonnike — APPROVED** [review]

File: README.MD (hunk 3 above)
Comment (jul-sh) [reply README.MD:19]: Fair point!

**Pull request merged**